src/picasso-definition/components/axes/index.js

```diff
@@ -9,6 +10,13 @@ export default function createAxes({ models, flags }) {
 
   const style = themeService.getStyles();
   const viewHandler = chartModel.query.getViewHandler();
+  const trackBy = (node, i) => {
+    if (i === 0) return 'axis';
```

Comment:
I think we should try to be more consistent around our if/else statements. Always write it with `{}`. I will take a look if we can enforce it through prettier or something (I've seen we already have different styles spread out it the code...).

Reply:
my bad. It won't happen again!

Reply:
I will go through the code and fix these if else style issues

Reply:
No worries! It's not the whole world, just a detail that looks nicer/cleaner

Reply:
I just fix that issue in the whole scatter plot. Then I realize I was doing it in the mini chart branch :D
So it will be fixed later when we merge mini chart!